src/main/java/org/jabref/cli/CliOptions.java

```diff
@@ -187,6 +187,7 @@ public String getCheckConsistencyOutputFormat() {
         return commandLine.getOptionValue("check-consistency-output-format", "TXT");
     }
 
+    @SuppressWarnings("checkstyle:EmptyLineSeparator")
```

Comment:
Checkstyle rules are there to enfore consistency, not to ingore them!

Reply:
Thanks, i get it.